src/api/app/assets/javascripts/webui/request.js

```diff
@@ -135,6 +135,9 @@ $(document).ready(function(){
     }
   });
   // TODO: Remove the enclosing code when the request_show_redesign feature is finished - END
+  $('#changes-item').on('shown.bs.tab', function () {
+    loadChanges();
+  });
```

Comment:
It seems this is a leftover now, `#changes-item` is gone since 933a14202dd262343223398f0cb8fab0f64b97d4

Reply:
Removed. Thanks!